Resources/Prototypes/_NF/Shipyard/spectre.yml

```diff
@@ -0,0 +1,31 @@
+- type: vessel
+  id: Spectre
+  name: Spectre
+  description: A large research mothership designed to be flown nexto a small fleet of other ships including salvage and food services.
+  price: 95000
```

Comment:
the anomaly spawner should be adding like 75k minimum to this price point. Since this is a special case thsed, and the potentiais ship to be so science-focused, and the potential for abuse of anomalies, please make this ship valued closer to 200k.